versions.yml

```diff
@@ -202,6 +222,13 @@ modGroups:
       url: https://mediafilez.forgecdn.net/files/4745/844/ShoulderSurfing-Fabric-1.20.1-2.5.jar
       requires: [Forge Config API Port]
 
+    - name: Blur
```

Comment:
Probably a HUD-Menu enchancement?

Reply:
Seems like Blur was under Misc previously, but I can put it under HUD enchancement if that's better.